src/shared/ContentsTable/TableEntries/PullEntries/PullDirEntry.spec.tsx

```diff
@@ -125,18 +137,20 @@ describe('PullDirEntry', () => {
     const dir = screen.getByText('dir')
     await user.hover(dir)
 
+    const queryKey = queryClient.getQueriesData({})?.at(0)?.at(0) as Array<any>
```

Comment:
Not really sure what's happening here but you can do as `Array<string>` and avoid any